### PR TITLE
Document curl usage with example

### PR DESCRIPTION
**Pull request by tristanlee85**
**NOTICE FOR DEPLOY TARGETS**

`main` is the (default) target branch for deploying to https://docs.edg.io relating to `@edgio` v5.x.

`main_v4` is the target branch for deploying to https://docs.layer0.co relating to `@layer0` 4.x.

If your change request applies to both versions, please create a pull request for both target branches.

-- You may remove this message --

### CODE REVIEW

File: src/pages/guides/develop/cli.md [hunk 1]
```diff
@@ -91,6 +91,28 @@ Creates a script that provides autocompletion for {{ PRODUCT_NAME }} CLI command
...
+| `--show-body` | Whether to output body in response.                                                                                                              |
+| `--help`      | Shows additional (less common) options.                                                                                                          |
+
+#### Example {/*example*/}
+
+```bash
+{{ FULL_CLI_NAME }} curl https://docs.edg.io
+```
+
+![{{ FULL_CLI_NAME }} curl example](/images/cli/curl.png)
```
Comment (jsernaque) [inline src/pages/guides/develop/cli.md:114]: Try to convert this to a code block.

File: src/pages/guides/develop/cli.md [hunk 2]
```diff
@@ -91,6 +91,28 @@ Creates a script that provides autocompletion for {{ PRODUCT_NAME }} CLI command
...
 
+### curl {/*curl*/}
+
+Uses `curl` to make a request to the specified URL, highlighting Edgio-specific telemetry information. See [Response](/guides/performance/response) for more information around response headers and telemetry values.
+
+#### Options {/*options*/}
+
+| Name          | Description                                                                                                                                      |
+|---------------|--------------------------------------------------------------------------------------------------------------------------------------------------|
+| `--json`      | Return JSON format only                                                                                                                          |
```
Comment (jsernaque) [inline src/pages/guides/develop/cli.md:102]: What is the default behavior (i.e., if this option is not passed)?

File: src/pages/guides/develop/cli.md [hunk 3]
```diff
@@ -91,6 +91,28 @@ Creates a script that provides autocompletion for {{ PRODUCT_NAME }} CLI command
...
+| `--debug`     | Run in verbose mode. This has no effect when using with `--json` option.                                                                         |
+| `--save-body` | Whether to save the response body. This will output tmp file path where the body was stored. This has no effect when using with `--json` option. |
+| `--show-body` | Whether to output body in response.                                                                                                              |
```
Comment (jsernaque) [inline src/pages/guides/develop/cli.md:105]: What is the default behavior (i.e., if this option is not passed)?

File: src/pages/guides/develop/cli.md (hunk 2 above)
Comment (tristanlee85) [reply src/pages/guides/develop/cli.md:102]: It's just the standard curl output

File: src/pages/guides/develop/cli.md (hunk 3 above)
Comment (tristanlee85) [reply src/pages/guides/develop/cli.md:105]: The response body isn't shown by default - just the headers and telemetry information.

Comment (jsernaque) [reply src/pages/guides/develop/cli.md]: How about this?
Includes the response body within the output.

File: src/pages/guides/develop/cli.md (hunk 2 above)
Comment (jsernaque) [reply src/pages/guides/develop/cli.md]: How about this?
Returns the output in JSON format.

**Pull request merged**